sql3/errors.go

```diff
@@ -706,6 +707,13 @@ func NewErrInsertValueOutOfRange(line, col int, columnName string, rowNumber int
 	)
 }
 
+func NewErrUnexpectedTimeQuantumTupleLength(line, col int, columnName string, rowNumber int, badValue []interface{}, length int) error {
```

Comment:
Unexpected Time Quantum Tuple Length

value `%v` out of range

I don't feel like these describe the same problem?

Reply:
oops - copy pasta